### ISSUE TITLE
v0.3.6 does not have the `dist` folder

### ISSUE THREAD

**Issue by MuTsunTsai** [jridgewell/gen-mapping#14]
Apparently there's something wrong with the release. Please fix it.

Update:
For those having the same issue, override `@jridgewell/gen-mapping` dependency to 0.3.5 will fix it for now.

**Comment by dinhquochan** [comment]
@jridgewell please check it.

**Comment by the-pawn-2017** [comment]
when I build my project, It's tell me:
```Cannot find module '***/node_modules/.pnpm/@ampproject+remapping@2.3.0/node_modules/@jridgewell/gen-mapping/dist/gen-mapping.mjs' imported from **/node_modules/.pnpm/@ampproject+remapping@2.3.0/node_modules/@ampproject/remapping/dist/remapping.mjs```
when I check the `node_modules`, there are no dist folder in `@ampproject/remapping`

**Comment by YUN-RU-TSENG** [comment]
I had problem too. Cannot find module '/node_modules/@jridgewell/gen-mapping/dist/gen-mapping.umd.js'

**Comment by E1fs** [comment]
Fail to build with github action now :( 
fix it plz

**Comment by dinhquochan** [comment]
Can't initial a Nuxt project or a TailwindCSS project yet :(

**Comment by ceynri** [comment]
If use pnpm, can overrides dependency version temporarily

```json
// package.json
{
  "pnpm": {
    "overrides": {
      "@jridgewell/gen-mapping": "0.3.5"
    }
  },
}
```

**Comment by leowux** [comment]
Please fix it.

**Comment by doublepithre** [comment]
Adding a override to version `0.3.5` in `package.json` works

```
"overrides": {
    "@jridgewell/gen-mapping": "0.3.5"
 }
```

**Comment by vandeace** [comment]
> Adding a override to version `0.3.5` in `package.json` works
> 
> ```
> "overrides": {
>     "@jridgewell/gen-mapping": "0.3.5"
>  }
> ```

where to put these code?

**Comment by zhlee1997** [comment]
> > Adding a override to version `0.3.5` in `package.json` works
> > ```
> > "overrides": {
> >     "@jridgewell/gen-mapping": "0.3.5"
> >  }
> > ```
> 
> where to put these code?

I did it in `package.json` and it works

**Comment by asself** [comment]
Has the intern started working today?

**Comment by dinhquochan** [comment]
> Adding a override to version `0.3.5` in `package.json` works
> 
> ```
> "overrides": {
>     "@jridgewell/gen-mapping": "0.3.5"
>  }
> ```

Thank you so much. It works for now.

**Comment by doublepithre** [comment]
> > Adding a override to version `0.3.5` in `package.json` works
> > ```
> > "overrides": {
> >     "@jridgewell/gen-mapping": "0.3.5"
> >  }
> > ```
> 
> where to put these code?

In the root level of `package.json` 

![image](https://github.com/user-attachments/assets/fa2aa75d-10fa-489c-96b3-83ace339d6a2)

**Comment by NoCrypt** [comment]
![image](https://github.com/user-attachments/assets/050455ec-c92d-49e6-80ef-4e649abaeac2)


literally [xkcd's dependency meme](https://xkcd.com/2347/)

**Comment by MichaelXF** [comment]
Owner should revert to 0.3.5:

```
npm dist-tag add @jridgewell/gen-mapping@0.3.5 latest
```

**Comment by fla0** [comment]
Тестируй свой говнокод лучше в следующий раз, процессы ломаешь у нормальных людей

**Comment by anthony-futures** [comment]
Still have this error with overrides : 

`Error: [BABEL]: Cannot find module '.../node_modules/@jridgewell/gen-mapping/dist/gen-mapping.umd.js' (While processing: .../node_modules/@babel/preset-env/lib/index.js)`


![image](https://github.com/user-attachments/assets/1a855ca2-ee2c-44d7-8bef-add5eb5f47ef)

An idea ?

**Comment by fla0** [comment]
Do someone has a phone of this guy?

**Comment by NoCrypt** [comment]
To override a package version, please see the docs for your package manager.

Bun: https://bun.sh/docs/install/overrides
Pnpm: https://pnpm.io/package_json#pnpmoverrides
Npm: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides
Yarn: https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/

**Comment by E1fs** [comment]
> Still have this error with overrides :
> 
> `Error: [BABEL]: Cannot find module '.../node_modules/@jridgewell/gen-mapping/dist/gen-mapping.umd.js' (While processing: .../node_modules/@babel/preset-env/lib/index.js)`
> 
> ![image](https://private-user-images.githubusercontent.com/118450747/394180719-1a855ca2-ee2c-44d7-8bef-add5eb5f47ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4MTgzMzgsIm5iZiI6MTczMzgxODAzOCwicGF0aCI6Ii8xMTg0NTA3NDcvMzk0MTgwNzE5LTFhODU1Y2EyLWVlMmMtNDRkNy04YmVmLWFkZDVlYjVmNDdlZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMFQwODA3MThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMWRmMTExMmU1ZDZkYWZjOGVmMDNlZDhhYThiYmQ5YzU3ZTJkNDdmNWI5ZDI3ZjdlMTEyM2IxM2IyYzUxM2YzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-6IJJHU7C-FihcfNPMdN8in_X-w3RwJR4yAjReZWCPs)
> 
> An idea ?

You used Yarn?
See this [Yarn resolutions](https://yarnpkg.com/configuration/manifest#resolutions)

**Comment by FoskyM** [comment]
It's truly a major accident.

**Comment by k-nero** [comment]
> Pnpm: https://pnpm.io/package_json#pnpmoverrides
> Npm: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides
> Yarn: [https://classic.yarnp](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/)

Bun does not support nested deps overrides,  you may need to use npm or yarn as temporary package manager

**Comment by s956142** [comment]
"overrides": {
    "@jridgewell/gen-mapping": "0.3.5"
  },
  "resolutions": {
    "@jridgewell/gen-mapping": "0.3.5"
  },

**Comment by anthony-futures** [comment]
> To override a package version, please see the docs for your package manager.
> 
> Bun: https://bun.sh/docs/install/overrides Pnpm: https://pnpm.io/package_json#pnpmoverrides Npm: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides Yarn: https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/

It's worked for me, I didn't know resolutions, thank you !!

**Comment by jridgewell** [comment]
I unpublished the broken version. Please rerun your package manager to pick up the previous `v0.3.5`

**Comment by NoCrypt** [comment]
> > Pnpm: https://pnpm.io/package_json#pnpmoverrides
> > Npm: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides
> > Yarn: [https://classic.yarnp](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/)
> 
> Bun does not support nested deps overrides, you may need to use npm or yarn as temporary package manager

`package.json`
```json
"overrides": {
  "@jridgewell/gen-mapping": "0.3.5"
},
```
This works fine on bun

**Comment by dazaooo** [comment]
> ![image](https://private-user-images.githubusercontent.com/57245077/394179259-050455ec-c92d-49e6-80ef-4e649abaeac2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4MTkwNjAsIm5iZiI6MTczMzgxODc2MCwicGF0aCI6Ii81NzI0NTA3Ny8zOTQxNzkyNTktMDUwNDU1ZWMtYzkyZC00OWU2LTgwZWYtNGU2NDlhYmFlYWMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEwVDA4MTkyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJmMGVhZWJhN2ViNjhhNjRhYzdhYTM1MzVhYTE3ZDYwYmFlMDdlY2MyNzgwZjNjOTc2MjI3MGRiY2MwZjM5MGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.XPIlKgIl7ji7CLhmPLzq3P4pZ9uCPzz843_FzwCEtN0)
> 
> literally [xkcd's dependency meme](https://xkcd.com/2347/)

😂best

**Comment by dennybiasiolli** [comment]
Thank you @jridgewell for fixing this so fast, everything is working now 😉

**Comment by efthimispegas** [comment]
Hello @jridgewell I see that you have not made a release of v.0.3.7 yet although you have mentioned in other duplicate issues that v.0.3.7 is out 

https://github.com/jridgewell/gen-mapping/issues/18#issuecomment-2530804024 

I would strongly suggest you not state information that is not valid, as many developers in the community have been thrown by these wrong statements.

Also, I would like to state that issue is still valid in many organizations where the public npmjs registry is not used and private artifactories are used instead. There the latest version (0.3.6) of `@jridgewell/gen-mapping` package is cached, causing the same issue. Currently, until a new release is out only workaround for resolving this issue is

```
// package.json
{
  //...
  "resolutions": {
    //...
    "@jridgewell/gen-mapping": "0.3.5"
  },
} 
```

**Comment by rusanz** [comment]
Same problem in our organization, if you can publish 0.3.5 with 0.3.6 tag will be perfect. We have a big problem with pipelines in CICD with this library. We need a solution asap.

**Comment by RubTalha** [comment]
https://stackoverflow.com/a/79272487/13086128

**Comment by MuTsunTsai** [comment]
> Hello @jridgewell I see that you have not made a release of v.0.3.7 yet although you have mentioned in other duplicate issues that v.0.3.7 is out

He really did release 0.3.7, I can testify that. But he probably removed it later for some reason, as I don't see 0.3.7 on NPM at this moment as well.

Anyway, he released 0.3.8 instead. Should solve the problem.
